Answers.md

```diff
@@ -0,0 +1,10 @@
+1. Semantic HTML is a way to make the anchors and tags of your website more "descriptive" so it's meta-data can be more easily found and comprehended. Instead of just <div> everywhere, semantic tags like <header> <section> <article> <footer> <figure> more accurately describe the "meaning" of your content, not just what goes where. 
+
```

Comment:
Yup. Makes sure we can kind of see the layout of the page better without having to actually have our browser render it. 

Also remember that it helps with the letting the "Google bots" get an idea of what our page looks like without having to load it up